mev_inspect/classifiers/helpers.py

```diff
@@ -84,3 +84,19 @@ def _filter_transfers(
         filtered_transfers.append(transfer)
 
     return filtered_transfers
+
+
+def get_amount_transferred_to_address(address: str, transfers: List[Transfer]) -> int:
+    for transfer in transfers:
+        if transfer.to_address == address:
```

Comment:
what if there are multiple transfers to the address? also what if they're the same tokens or different